README.md

```diff
@@ -36,6 +36,10 @@ Describe bugs, small features, and refactors like you would to a junior develope
 * 👀 Addresses developer replies & comments on its PRs
 * 🔎 Uses embedding-based code search
 
+### Why Sweep is Unique 🦄
```

Comment:
move this below the demo. also remove the last sentence. Address copilot and chatgpt separately